src/components/Map/filter/filterOperators.ts

```diff
@@ -1,75 +1,46 @@
-import { Filter, FilterAddition, HighlightId } from './types'
+import type {
+  Filter,
+  FilterAddition,
+  FilterExpression,
+  FilterTail,
+  HighlightId,
+} from "./types";
 
 export interface FilterProperties {
-  id?: string
-  forLayer?: string
+  id?: string;
+  forLayer?: string;
 }
 
-/**
```

Comment:
I've removed these filters for now – they do address a need but are unused. Something similar will appear again when I'll work on the upcoming elevator highlighting task.